packages/angular_devkit/build_angular/src/webpack/configs/common.ts

```diff
@@ -434,12 +438,26 @@ export async function getCommonConfig(wco: WebpackConfigOptions): Promise<Config
             name: 'vendor',
             chunks: (chunk) => chunk.name === 'main',
             enforce: true,
-            test: /[\\/]node_modules[\\/]/,
+            test: VENDORS_TEST,
           },
         },
       },
     },
-    plugins: [new NamedChunksPlugin(), new DedupeModuleResolvePlugin({ verbose }), ...extraPlugins],
+    plugins: [
+      new NamedChunksPlugin(),
+      new DedupeModuleResolvePlugin({ verbose }),
+      new DevToolsIgnorePlugin({
```

Comment:
I feel it would be better to just not add the plugin when `vendorSourceMap` is truthy, Instead of the options.

Ex: https://github.com/angular/angular-cli/blob/56320245a126481816a8ccfb0cc248a00be49561/packages/angular_devkit/build_angular/src/webpack/configs/common.ts#L222-L228

Reply:
Makes sense! Might gain us a bit of a speed bump if the tapping isn't zero cost.